src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -142,4 +150,17 @@ private function extractCallable(Route $route)
         } catch (\InvalidArgumentException $e) {
         }
     }
+
+    private function findRouteNameContaining(string $name, RouteCollection $routes)
+    {
+        $foundRoutesNames = array();
+        foreach ($routes as $routeName => $route) {
+            if (false === stripos($routeName, $name)) {
```

Comment:
Maybe can you invert the test, and put `$foundRoutesNames[] = $routeName;` in the if block to save 1 line (nitpicking).